users/users_test.go

```diff
@@ -51,6 +51,20 @@ func TestValidate(t *testing.T) {
 			},
 			err: users.ErrMalformedEntity,
 		},
+		"validate user with utf8 email (cyrillic)": {
+			user: users.User{
+				Email:    "почта@кино-россия.рф",
+				Password: password,
+			},
+			err: nil,
+		},
+		"validate user with utf8 email (hieroglyph)": {
+			user: users.User{
+				Email:    "艾付忧西开@艾付忧西开.再得",
+				Password: password,
+			},
+			err: nil,
+		},
```

Comment:
Thank you for your help! If have time and want to make happy our codecover you could add here 3 tests for invalid host, invalid domain and invalid extension.